configs/systemd_common_config

```diff
@@ -0,0 +1,35 @@
+#
+# Resize the data partition to fill the remaining space, using parted, with systemd
+#
+function grow_data_partition() {
+    log_info "Adding systemd init script to run parted and resize the data partition on boot"
+    log_info "to fill all the available space on the storage media"
+    run_and_log_cmd "sudo mkdir -p work/rpi/etc/systemd/system/"
+    run_and_log_cmd "sudo mkdir -p work/rootfs/etc/systemd/system/data.mount.wants/"
+    cat <<- EOF > work/rpi/etc/systemd/system/mender-grow-data.service
+		  [Unit]
+		  Description=Mender service to grow data partition size
+		  DefaultDependencies=no
+		  Before=data.mount
+		  Before=systemd-growfs@data.service
+
+		  [Service]
+		  Type=oneshot
+		  User=root
+		  Group=root
+		  ExecStart=/sbin/parted --script $(disk_get_device_base "${MENDER_STORAGE_DEVICE_BASE}") resizepart ${MENDER_DATA_PART_NUMBER} 100%
+
+		  [Install]
+		  WantedBy=data.mount
+	EOF
+
+    # Install
+    run_and_log_cmd "cp work/rpi/etc/systemd/system/mender-grow-data.service \
+                      work/rootfs/etc/systemd/system/"
```

Comment:
Why was this done in `/rpi/` and then copied over? If the `/rpi/` is a vendor specific requirement for Raspberry Pi images, we should keep it for such boards but in any case removed from the generic.